apps-rendering/src/relatedContent.ts

```diff
@@ -23,6 +22,7 @@ import {
 	isLabs,
 	isLetter,
 	isLive,
+	isReview,
```

Comment:
This is the function mentioned previously that checks for any of these tags: `tone/review`, `tone/livereview`, `tone/albumreview`